src/Lessons-api.Data/Models/LessonEntity.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+
+namespace Lessons_api.Data.Models
+{
+    public class LessonEntity
+    {
+        public LessonEntity()
+        {
+            StudentLessonRels = new HashSet<StudentLessonRel>();
+        }
+
+        public int Id { get; set; }
+        public int TeacherId { get; set; }
```

Comment:
If you have `TeacherId` in Lesson you need also add navigation property for that